python/tk_multi_workfiles/my_tasks/my_tasks_model.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, project, user, extra_display_fields, parent, bg_task_manager=
         """
         self.extra_display_fields = extra_display_fields or []
         filters = [["project", "is", project],
-                   ["task_assignees", "is", user]]
+                   ["task_assignees", "is", user],
```

Comment:
@jfboismenu, great point, might as well give more control. Do you think we should use the same syntax as the filters for the other tabs? https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/master/info.yml#L109